app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryPreferencesEducator.tsx

```diff
@@ -0,0 +1,195 @@
+            bucket.preferenceShortCode &&
+            bucket.preferenceShortCode.includes("T2") &&
+            !bucket.preferenceShortCode.includes("V") &&
+            bucket.totalSubmittedApps
+        )
+        .map((bucket, index, arr) => (
+          <LotteryPreferences
+            unitsAvailable={bucket.unitsAvailable}
+            totalSubmittedApps={bucket.totalSubmittedApps}
+            preferenceName={bucket.preferenceName.replace("Tier 2 ", "")}
+            numVeteranApps={
+              veteranAppsByPreferenceMap[bucket.preferenceShortCode.replace("-", "-V-")] ?? null
+            }
+            lastPref={arr.length - 1 === index}
+          />
+        ))}
+      {!isEducatorOne && (
+        <>
+          <header className="border-b border-gray-450 my-4">
+            <Heading styleType="underlineWeighted" className="mx-8 text-xs" priority={2}>
+              {t("lottery.generalPublic")}
+            </Heading>
+          </header>
+          {lotteryBucketsDetails.lotteryBuckets
+            .filter(
+              (bucket) =>
+                bucket.preferenceShortCode &&
+                !bucket.preferenceShortCode.includes("T1") &&
+                !bucket.preferenceShortCode.includes("T2") &&
+                !bucket.preferenceShortCode.includes("V") &&
+                bucket.totalSubmittedApps
+            )
+            .map((bucket, index, arr) => (
+              <LotteryPreferences
+                unitsAvailable={bucket.unitsAvailable}
+                totalSubmittedApps={bucket.totalSubmittedApps}
+                preferenceName={bucket.preferenceName}
+                numVeteranApps={
+                  veteranAppsByPreferenceMap[bucket.preferenceShortCode.replace("-", "-V-")] ?? null
+                }
+                lastPref={arr.length - 1 === index}
+              />
+            ))}
+          <div className="bg-gray-100 border-b mb-4 p-4">
+            <hr className="border-b-4 border-primary" />
+          </div>
+          {lotteryBucketsDetails.lotteryBuckets
+            .filter((bucket) => bucket.preferenceName === "generalLottery")
+            .map((bucket) => (
+              <React.Fragment key={bucket.preferenceOrder}>
+                <div className="px-8">
+                  <Heading
+                    className="font-sans font-semibold text-sm tracking-wide uppercase"
+                    priority={3}
+                  >
+                    {t("lottery.generalPool")}
+                  </Heading>
+                  <p className="mb-1 text-sm">{t("lottery.anyRemainingUnits")}</p>
+                  <p className="text-gray-700 text-sm">
+                    {t("lottery.numberApplicantsQualifiedForGeneralPool", {
+                      number: bucket.totalSubmittedApps,
+                    })}
+                  </p>
+                </div>
+                <hr className="mt-4" />
+              </React.Fragment>
+            ))}
+        </>
+      )}
+    </div>
+  )
+}
```

Comment:
suggestion: This is a general suggestion for the structure of this file, and just my personal opinion - all of the inline code could be broken up into a few more digestible chunks, e.g.

```js
// this just pseudocode

const LotteryPreferences = ({ ... })
const TierOneBuckets = ({ ... })
const TierTwoBuckets = ({ ... })
const GeneralBuckets = ({ ... })

export const ListingDetailsLotteryPreferencesEducator = ({
  // ...
  <TierOneBuckets ... />
  <TierTwoBuckets ... />
  {!isEducatorOne && <GeneralBuckets ... />}
  // ...
})
```